recipes/osx-pocl-opencl/meta.yaml

```diff
@@ -0,0 +1,22 @@
+package:
+  name: osx-pocl-opencl
+  version: 1
+
+build:
+  number: 0
+  track_features:
+    - osx_pocl_opencl
+  skip: True  # [not osx]
+
+test: {}
+
+about:
+  home: http://github.com/conda-forge/osx-pocl-opencl-feedstock
+  license: MIT
+  license_family: MIT
```

Comment:
I think this defaults to BSD 3-Clause because the feedstock will be under BSD 3-Clause.